modules/loadbalancer/main.tf

```
@@ -173,9 +174,9 @@ resource "azurerm_network_security_rule" "allow_inbound_ips" {
   direction                   = "Inbound"
   access                      = "Allow"
   protocol                    = title(replace(each.value.protocol, "all", "*"))
-  description                 = "Auto-generated for load balancer ${var.name} port ${each.value.protocol}/${each.value.port}: allowed inbound IP ranges"
+  description                 = "Auto-generated for load balancer ${var.name} port ${each.value.protocol}/${coalesce(each.value.backend_port, each.value.port)}: allowed inbound IP ranges"
```

```suggestion
  description                 = "Auto-generated for load balancer ${var.name} port ${each.value.protocol}/${coalesce(try(each.value.rule.backend_port, null), each.value.rule.port)}: allowed inbound IP ranges"
```

as `coalesce` does not work on non existing entities maybe we should use the same `coalesce(try())` combo also here

---

Actually it's already there. backend_port is set to null with a try condition in the output_rules, and we are looping over output_rules.

---

👍🏻